sspl_test/run_qa_test.sh

```diff
@@ -194,6 +211,16 @@ execute_test()
     $sudo $script_dir/run_sspl-ll_tests.sh $plan
 }
 
+if [ "$SSPL_STORE_TYPE" == "confstor" ]
+then
+    # Read common key which are needed to fetch confstor config.
+    machine_id=`cat /etc/machine-id`
+    minion_id=`conf $common_config get "cluster>server_nodes>$machine_id"`
```

Comment:
minion id is no more recommended to be used. but from machineid-to-nodename mapping, which will have value as srvnode-1, etc , same as we had in minion-id. @sandeepsgt @shriya-deshmukh   we clarified this earlier, please incorporate accordingly

Reply:
Internally I am using machine id only to map the node name, just used minion-id as variable name  , 
renamed it to srvnode in latest commit.